Lib/test/test_gc.py

```diff
@@ -1058,7 +1058,18 @@ class Z:
         callback.assert_not_called()
         gc.enable()
 
+
+class IncremetalGCTests(unittest.TestCase):
+
+    def setUp(self):
+        gc.enable()
+
+    def tearDown(self):
+        gc.disable()
```

Comment:
Won't this leave GC disabled when the last test in this class is done?  Subsequent tests would then run with GC disabled, right?  If that's true, then perhaps there should be a `tearDownClass()` that re-enables GC one last time.

Reply:
This module disables gc [during setup](https://github.com/python/cpython/blob/main/Lib/test/test_gc.py#L1481) and re-enables it at the end, so this (and the `setUp` method) just turns GC back on this for this test.

Reply:
I suggest leaving a comment to that effect.  _(we often see code in huge files like this in bits and pieces so would otherwise not realize this in things like this diff review context)_